seqr/views/utils/orm_to_json_utils.py

```diff
@@ -757,10 +757,30 @@ def get_json_for_matchmaker_submission(submission):
         additional_model_fields=['contact_name', 'contact_href', 'submission_id'])
 
 
-def get_json_for_rna_seq_outliers(models, **kwargs):
-    additional_values = {
-        'isSignificant': Case(
-            When(p_adjust__lt=models.model.SIGNIFICANCE_THRESHOLD, then=Value(True)), default=Value(False)
-        ),
-    }
-    return get_json_for_queryset(models, additional_values=additional_values, **kwargs)
+def get_json_for_rna_seq_outliers(filters, significant_only=True):
+    filters['sample__is_active'] = True
+    data_by_individual_gene = defaultdict(lambda: defaultdict(lambda: defaultdict(list)))
+
+    for model, outlier_type in [(RnaSeqOutlier, 'outliers'), (RnaSeqSpliceOutlier, 'spliceOutliers')]:
+        significant_field = model.SIGNIFICANCE_FIELD
+
+        significant_filter = {f'{significant_field}__lt': model.SIGNIFICANCE_THRESHOLD}
```

Comment:
probably neater to use `model.SIGNIFICANCE_FIELD` directly here instead of assigning it to another variable first